.travis.yml

```diff
@@ -10,11 +10,11 @@ compiler:
 notifications:
   email:
     recipients:
-      # - user@email.com
+       - dave@dav.ee
 env:
   matrix:
-    - ROS_DISTRO="jade"  ROS_REP=ros              UPSTREAM_WORKSPACE=https://raw.githubusercontent.com/ros-planning/moveit_docs/jade-devel/moveit.rosinstall BEFORE_SCRIPT="echo 'testing'" TEST_BLACKLIST="moveit_core"
-    - ROS_DISTRO="jade"  ROS_REP=ros-shadow-fixed UPSTREAM_WORKSPACE=https://raw.githubusercontent.com/ros-planning/moveit_docs/jade-devel/moveit.rosinstall BEFORE_SCRIPT="echo 'testing'"
+    - ROS_DISTRO=kinetic ROS_REPO=ros              UPSTREAM_WORKSPACE=https://raw.githubusercontent.com/ros-planning/moveit/kinetic-devel/moveit.rosinstall BEFORE_SCRIPT="echo 'testing'" TEST_BLACKLIST="moveit_core"
+    - ROS_DISTRO=kinetic ROS_REPO=ros-shadow-fixed UPSTREAM_WORKSPACE=https://raw.githubusercontent.com/ros-planning/moveit/kinetic-devel/moveit.rosinstall BEFORE_SCRIPT="echo 'testing'"
```

Comment:
I probably messed up the rebasing. Maybe it would be cleaner if I just created a new PR? Since I only want to change the build type to debug.

The (new) failing assertion needs to be fixed before anything gets merged.